src/index.js

```diff
@@ -756,23 +756,66 @@ export class ULabel {
         });
     }
 
+    static handle_deprecated_arguments() {
+        return {
+            "container_id": arguments[0],  // Required
+            "image_data": arguments[1],    // Required
+            "username": arguments[2],      // Required
+            "submit_butons": arguments[3], // Required
+            "subtasks": arguments[4],      // Required
+            "task_meta": arguments[5] ?? null,       // Use default if optional argument is undefined
+            "annotation_meta": arguments[6] ?? null, // Use default if optional argument is undefined
+            "px_per_px": arguments[7] ?? 1,          // Use default if optional argument is undefined
+            "initial_crop": arguments[8] ?? null,    // Use default if optional argument is undefined
+            "initial_line_size": arguments[9] ?? 4,  // Use default if optional argument is undefined
+            "config_data": arguments[10] ?? null,    // Use default if optional argument is undefined
+            "toolbox_order": arguments[11] ?? null   // Use default if optional argument is undefined
+        }
+    }
+
     // ================= Construction/Initialization =================
 
-    constructor(
-        container_id,
-        image_data,
-        username,
-        submit_buttons,
-        subtasks,
-        task_meta = null,
-        annotation_meta = null,
-        px_per_px = 1,
-        initial_crop = null, //{top: #, left: #, height: #, width: #,}
-        initial_line_size = 4,
-        instructions_url = null,
-        config_data = null,
-        toolbox_order = null
-    ) {
+    constructor(kwargs) {
+        // Ensure arguments were recieved
+        if (arguments.length === 0) {
+            console.error("ULabel was given no arguments")
+        }
+        // The old constructor took in up to 11 arguments, so if more than 1 argument is present convert them to the new format
+        else if (arguments.length > 1) {
```

Comment:
So `Object.length` would return `undefined` and thus not trigger this when kwargs are passed in, correct? I suppose that is one way of detecting old arguments that probably works, although maybe just checking the type is clearer